interface/billing/billing_report.php

```diff
@@ -260,10 +260,10 @@ function placeModal(Message) {
             let dModal = "<div class='modal fade' id='confirmDialog' aria-hidden='true'><div class='modal-dialog'><div class='modal-content'>" +
```

Comment:
The expand option here is broken:
![image](https://user-images.githubusercontent.com/46497068/92337048-bca4a200-f074-11ea-9591-bd7eeb189c44.png)

Reply:
@bradymiller How does this darn thing work? xD

Reply:
I think if no data the expand function can't expand.

cc @bradymiller

Reply:
In my opinion you really don't need an expand function here (that is unless there's a lot of content and it could break it) @bradymiller

Reply:
Hi @stu01509 and @tywrenn ,
The expand/collapse is for showing things that have happened with that specific bill:

![image](https://user-images.githubusercontent.com/278968/93565903-f7de8500-f940-11ea-8609-154bc1368d02.png)

![image](https://user-images.githubusercontent.com/278968/93565926-0331b080-f941-11ea-8aa4-f4f3f294096d.png)

Reply:
Hi @bradymiller 

So, If that without any data, the expand and collapse will not trigger, right?